src/main/java/fi/vm/yti/datamodel/api/v2/endpoint/ClassController.java

```diff
@@ -209,6 +209,14 @@ private ResourceInfoBaseDTO handleGetClassOrNodeShape(String prefix, String clas
         return dto;
     }
 
+    @Operation(summary = "Check if identifier for resource already exists")
+    @ApiResponse(responseCode = "200", description = "Boolean value indicating whether prefix")
+    @GetMapping(value = "/{prefix}/freeIdentifier/{identifier}", produces = APPLICATION_JSON_VALUE)
```

Comment:
Should we use kebab case in the endpoints as discussed earlier?

Reply:
Changed to kebab case.

Changed datamodel freePrefix endpoint to kebab case as well